01-introduction.Rmd

```diff
@@ -0,0 +1,204 @@
+
+### Configure two-factor authentication for the Analytical Platform
+
+When you sign in to the Analytical Platform for the first time, you will be prompted to configure additional two-factor authentication (2FA) for the Analytical Platform itself.
+
+To sign in, go to the Analytical Platform [control panel](https://cpanel-master.services.alpha.mojanalytics.xyz).
+
+You must configure 2FA for the Analytical Platform using a mobile app (on your personal or work phone). As described in Section \@ref(configure-two-factor-authentication-for-github), we recommend that you use __Authy__.
+
+## Access the Analytical Platform
+
+The main entry point to the Analytical Platform is the [control panel](https://cpanel-master.services.alpha.mojanalytics.xyz). From here, you can access RStudio and JupyterLab and can manage your warehouse data sources.
+
+You may also need to access other services and tools on the Analytical Platform:
+
+*   [Airflow](https://airflow.tools.alpha.mojanalytics.xyz)
+*   [AWS console](https://aws.services.alpha.mojanalytics.xyz)
+*   [Concourse](https://concourse.services.alpha.mojanalytics.xyz)
+*   [Grafana](https://grafana.services.alpha.mojanalytics.xyz)
+*   [Kibana](https://kibana.services.alpha.mojanalytics.xyz)
+
+
+## Deploy RStudio, JupyterLab and Airflow
+
+Before using RStudio, JupyterLab and Airflow, you must first deploy them.
+
+ To deploy RStudio, JupyterLab and Airflow on the Analytical Platform, you should complete the following steps:
+
+1.  Go the Analytical Platform [control panel](https://cpanel-master.services.alpha.mojanalytics.xyz).
+2.  Select the __Analytical tools__ tab.
+3.  Select the __Deploy__ buttons next to RStudio, JupyterLab and Airflow.
+
+It may take a few minutes for the tools to deploy.
+
+## Open RStudio, JupyterLab or Airflow
+
+To open RStudio, JupyterLab or Airflow:
+
+1.  Go the Analytical Platform [control panel](https://cpanel-master.services.alpha.mojanalytics.xyz).
+2.  Select the __Analytical tools__ tab.
+3.  Select the __Open__ button to the right of the tool's name.
+
+## Restart RStudio, JupyterLab or Airflow
+
+If your RStudio, JupyterLab or Airflow is not working as expected, it may help to restart it.
+
+To restart RStudio, JupyterLab or Airflow:
+
+1.  Go the Analytical Platform [control panel](https://cpanel-master.services.alpha.mojanalytics.xyz).
+2.  Select the __Analytical tools__ tab.
+3.  Select the __Restart__ button to the right of the tool's name.
+
+## Unidle Rstudio, JupyterLab or Airflow
+
+If your RStudio, JupyterLab or Airflow instance is inactive for an extended period of time (e.g., overnight) it will be idled.
+
+1.  Go the Analytical Platform [control panel](https://cpanel-master.services.alpha.mojanalytics.xyz).
+2.  Select the __Analytical tools__ tab.
+3.  Select the __Unidle__ button to the right of the tool's name. 
+
+Unidling usually only takes a few seconds, however, it can take up to several minutes.
```

Comment:
```suggestion
Unidling usually only takes a few seconds, however it can take up to several minutes.
```

Reply:
The original punctuation is consistent with the [Guardian style guide](https://www.theguardian.com/guardian-observer-style-guide-a).